library/Vanilla/UploadedFile.php

```diff
@@ -245,7 +247,7 @@ private function tryApplyImageProcessing(): void {
             "width" => $width ?? 0,
         ];
 
-        if (\Gdn::config("ImageUpload.Limits.Enabled")) {
+        if (\Gdn::config("ImageUpload.Limits.Enabled") && !$this->getBypassUploadLimits()) {
```

Comment:
I would probably be happier to see this class have a basic property called '$maxImageWidth`. Set this in the constructor from the config with a value of `null` meaning there is no max width. That way you can just set this value to whatever. It allows developers to do the following:

1. Set a different width.
2. Set no enforcement.
3. It's clearer that the limit is just on image widths which also allows us to set other limits in the future.
4. It does this all with one property and not a strange combination of config and properties.